File: cs50x/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Get the input height
    int height;
    do
    {
        height = get_int("Height: ");
    }
    // Input a positive integer between 1 and 8, inclusive.
    // If not, re-prompt the user until they cooperate.
    while (height > 8 || height < 1);

    // Build the mario block
    for(int i = 1; i <= height; i++)
    {
        // Build space
        int j;
        for (j = height - i; j > 0; j--)
        {
            printf(" ");
        }

        // Build hash sign #
        for (j = i; j > 0; j--)
        {
            printf("#");
        }

        // Build space in bettween hash sign
        printf("  ");

        // Build hash sign #
        for (j = i; j > 0; j--)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: cs50x/sentimental-mario-more/mario.py
# TODO
from cs50 import get_int

# Force the user to enter required integer
while True:
    times = get_int("Enter an integer for mario pyramid height: ")
    if times <= 0 or times > 8:
        print("Wrong input")
    elif times >= 1 or times <= 8:
        break

for i in range(1, times + 1):
    # print the space
    print(" " * (times - i), end="")
    print("#" * i, end="")
    print("  ", end="")
    print("#" * i, end="")
    print()